mtp_api/apps/core/tests/utils.py

```diff
@@ -58,20 +60,24 @@ def make_applications():
     owner = get_user_model().objects.first()
 
     def make_application_and_roles(client_id, name, *roles):
-        new_app, _ = Application.objects.get_or_create(
-            client_id=client_id,
-            client_type='confidential',
-            authorization_grant_type='password',
-            client_secret=client_id,
-            name=name,
-            user=owner,
+        app = Application.objects.get(
```

Comment:
won't this throw a `DoesNotExist`?

Reply:
Yup, just ran into that issue in CI! Once again I was using a kludge of SQLAlchemy and Django ORM query syntax